go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/Azure/go-autorest v14.2.0+incompatible // indirect
 	github.com/Azure/go-autorest/autorest v0.11.27 // indirect
 	github.com/Azure/go-autorest/autorest/adal v0.9.20 // indirect
-	github.com/openshift/api v0.0.0-20230213202419-42edf4f1d905
+	github.com/openshift/api v0.0.0-20220712151050-2647eb31dee7 // release-4.11
```

Comment:
Should we use `release-4.14` instead? Or the last released version, `release-4.12`?

Reply:
may be yes :)